client.go

```diff
@@ -857,6 +857,7 @@ func GetSDKV3ForTozIDUser(login TozIDLoginRequest) (*ToznySDKV3, error) {
 		return nil, err
 	}
 	// TODO: rework this to support brokered logins. See JS SDK for examples
```

Comment:
remove todo

Reply:
will do once all the changes are in place